sources/mysql/README.md

```diff
@@ -259,12 +264,12 @@ type, which supports UTF-8.
 As noted earlier when discussing [schema conversion of
 Spatial datatype](#spatial-datatype), Spanner does not support spatial datatypes and are
 mapped to `STRING(MAX)` Spanner type. Data conversion for spatial datatypes is different
-for both `mysql` and `mysqldump` driver.
+for both `mysql` direct connect and `mysqldump` file.
```

Comment:
Suggest rephrasing:

There are differences in data conversion for spatial datatypes depending on whether direct connect or a `mysqldump` file is used:

Reply:
Done, much cleaner. Thanks.